### ISSUE TITLE
Permissions exception when you don't have permissions in a workflow 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#13506]
We have an exception in the dotcms.log file every time you try to see a content as any user that don't have permissions over the assigned workflow. 

## Expected Behavior
We need catch this exception to avoid a lot of noise in the log file 

## Current Behavior
We are validating the permissions and we are not showing the actions if you don't have permissions, but we have this exception in the log.
ST: https://gist.github.com/bryanboza/dcb0091cb59839310662673932ea101b

## Steps to Reproduce (for bugs)
1. Create a new workflow 
2. Add a new action but set the who can use field just to admin user 
3. Add a new content type and set the created workflow 
4. Add full permissions to Chris publisher to this content type
5. Login as Chris Publisher and add a new content 
6. Open the content to edit and make sure you can't see the WF actions 

## Your Environment
Tested in the last master code // postgres // FF

**Comment by oswaldogallango** [comment]
@bryanboza this isn't a workflow issue. 
```
[01/02/18 12:01:22:866 EST] ERROR servlet.VelocityServlet: Exceptioncom.dotmarketing.exception.DotSecurityException: User Chris Publisher : dotcms.org.2795 does not have permission to read fdb3f906-e9c4-46c4-b7e4-148201271d04 @ http-nio-8080-exec-7
com.dotmarketing.exception.DotSecurityException: User Chris Publisher : dotcms.org.2795 does not have permission to read fdb3f906-e9c4-46c4-b7e4-148201271d04
	at com.dotmarketing.business.VersionableAPIImpl.findWorkingVersion_aroundBody0(VersionableAPIImpl.java:55) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotmarketing.business.VersionableAPIImpl$AjcClosure1.run(VersionableAPIImpl.java:1) ~[dotcms_4.3.0_ceff036.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotmarketing.business.VersionableAPIImpl.findWorkingVersion(VersionableAPIImpl.java:49) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.services.PageContextBuilder.populateContext(PageContextBuilder.java:93) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.services.PageContextBuilder.<init>(PageContextBuilder.java:63) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.services.PageContextBuilder.<init>(PageContextBuilder.java:70) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.servlet.VelocityEditMode.serve(VelocityEditMode.java:63) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.servlet.VelocityEditMode.serve(VelocityEditMode.java:76) ~[dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.servlet.VelocityServlet.service_aroundBody0(VelocityServlet.java:53) [dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.servlet.VelocityServlet$AjcClosure1.run(VelocityServlet.java:1) [dotcms_4.3.0_ceff036.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) [aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:43) [dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.aspects.interceptors.CloseDBMethodInterceptor.invoke(CloseDBMethodInterceptor.java:34) [dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.aspects.aspectj.CloseDBAspect.invoke(CloseDBAspect.java:42) [dotcms_4.3.0_ceff036.jar:?]
	at com.dotcms.rendering.velocity.servlet.VelocityServlet.service(VelocityServlet.java:32) [dotcms_4.3.0_ceff036.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:742) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:728) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:472) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:395) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.24]
	at org.apache.jsp.html.error.custom_002derror_002dpage_jsp._jspService(custom_002derror_002dpage_jsp.java:163) [ROOT/:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70) [jasper.jar:8.5.24]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:742) [servlet-api.jar:?]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:443) [jasper.jar:8.5.24]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:386) [jasper.jar:8.5.24]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:330) [jasper.jar:8.5.24]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:742) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:728) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:472) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:395) [catalina.jar:8.5.24]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.24]
	at org.apache.catalina.core.StandardHostValve.custom(StandardHostValve.java:395) [catalina.jar:8.5.24]
	at org.apache.catalina.core.StandardHostValve.status(StandardHostValve.java:254) [catalina.jar:8.5.24]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:177) [catalina.jar:8.5.24]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.24]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:650) [catalina.jar:8.5.24]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87) [catalina.jar:8.5.24]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342) [catalina.jar:8.5.24]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:803) [tomcat-coyote.jar:8.5.24]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) [tomcat-coyote.jar:8.5.24]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:790) [tomcat-coyote.jar:8.5.24]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1459) [tomcat-coyote.jar:8.5.24]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.24]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_112]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_112]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.24]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_112]

```
The problem is caused because there are these tags ````<img src="/html/js/lightbox/images/loading.gif">```` trying to load a gif image that doesn't exist in dotCMS anymore. The system after fail to load the file makes a call to the /home/404-error page and when the velocityServlet try to get the versionable of the Template used in that error page, the User doesn't have read permissions.  

That /html/js/lightbox/images/loading.gif file was removed long time ago with the lightbox library, but seems that there are some places that still have it like: src/main/webapp/WEB-INF/jsp/tag_manager/tag_manager.jsp,
src/main/webapp/html/portlet/ext/common/edit_permissions_tab_inc.jsp,  src/main/webapp/html/portlet/ext/common/edit_permissions_tab_inc_ajax.jsp and  src/main/webapp/html/portlet/ext/useradmin/view_users.jsp

**Comment by wezell** [comment]
Please replace the references to these with: 

/html/js/dojo/custom-build/dojox/widget/Standby/images/loading.gif

**Comment by oswaldogallango** [comment]
P.R https://github.com/dotCMS/core/pull/13746

**Comment by bryanboza** [comment]
Fixed, after the fix #9315 we don't have this error
